core/trino-spi/src/test/java/io/trino/spi/testing/InterfaceTestUtils.java

```diff
@@ -49,6 +50,9 @@ public static <I, C extends I> void assertAllMethodsOverridden(Class<I> iface, C
             if (method.getDeclaringClass() == Object.class) {
                 continue;
             }
+            if (Arrays.stream(method.getAnnotations()).anyMatch(annotation -> annotation instanceof Deprecated deprecated && deprecated.forRemoval())) {
```

Comment:
@electrum what do you think about this change?

Reply:
FTR, I removed this line, and will add any dependency methods that are found to be missing.